src/lib.rs

```diff
@@ -622,22 +622,28 @@ impl TOTP {
         if self.algorithm == Algorithm::Steam {
             host = "steam";
         }
-        let account_name: String = urlencoding::encode(self.account_name.as_str()).to_string();
-        let mut label: String = format!("{}?", account_name);
-        if self.issuer.is_some() {
-            let issuer: String =
-                urlencoding::encode(self.issuer.as_ref().unwrap().as_str()).to_string();
-            label = format!("{0}:{1}?issuer={0}&", issuer, account_name);
+        let account_name = urlencoding::encode(self.account_name.as_str()).to_string();
+        let mut params = vec![
+            format!("secret={}", self.get_secret_base32()),
+            format!("digits={}", self.digits),
+            format!("algorithm={}", self.algorithm),
```

Comment:
Digits and Algo could also not be present in case of default parameters

(6 and SHA-1)

Reply:
I'd be happy implement that as well in about half an hour.

Reply:
Added in 06627f2b26dcaf7752fe21e1597ad94d35441e04